### PR TITLE
ref(js): Convert CreateProject to a FC

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/projectInstall/createProject.tsx [hunk 1]
```diff
@@ -141,191 +227,49 @@ class CreateProject extends Component<Props, State> {
...
-  }
+        {createProjectForm}
+      </div>
+    </Fragment>
+  );
 }
 
-// TODO(davidenwang): change to functional component and replace withTeams with useTeams
-export default withRouteAnalytics(
-  withApi(withSentryRouter(withOrganization(withTeams(CreateProject))))
```
Comment (evanpurkhiser) [inline static/app/views/projectInstall/createProject.tsx:327]: This was the last usage of `withTeams`!

Comment (davidenwang) [reply static/app/views/projectInstall/createProject.tsx:327]: huge huge huge

File: static/app/views/projectInstall/createProject.tsx [hunk 2]
```diff
@@ -141,191 +227,49 @@ class CreateProject extends Component<Props, State> {
...
-);
 export {CreateProject};
```
Comment (davidenwang) [inline static/app/views/projectInstall/createProject.tsx:273]: should we just have this as a default export?

**Review by davidenwang — APPROVED** [review]

**Review by davidenwang — APPROVED** [review]

File: static/app/views/projectInstall/createProject.tsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/views/projectInstall/createProject.tsx:273]: I am trying to remove default exports

**Pull request merged**